tests/unit/test_viz.py

```diff
@@ -17,30 +17,30 @@
 import numpy as np
 from numpy.random import default_rng as rng
 
-from flamingpy.utils.viz import to_pi_string
+import flamingpy.utils.viz as viz
```

Comment:
Excellent work, @sduquemesa. Can you just briefly clarify why this change was needed here? Matter of better conventions?